components/dashboard/PortfolioSummary.tsx

```diff
@@ -85,7 +96,10 @@ const PortfolioSummary: FunctionComponent<PortfolioSummaryProps> = ({ title, dat
   const [showClaimModal, setShowClaimModal] = useState(false);
   const [showSuccessModal, setShowSuccessModal] = useState(false);
   
-  return data.length > 0 ? (
+  return (  
+    <Loading isLoading={loading} loadingType="skeleton" width="100%" height={300}>

```

Comment:
Why are you doing this ? We want to make a skeleton loading for this part not rendering it randomly 
![image](https://github.com/user-attachments/assets/d82fe119-7c42-404e-93f1-4995f2687f6d)